### PR TITLE
Readme update to be acccurate

### PR DESCRIPTION
**Pull request by victoriabernard92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ There are two types of configuration that `wrangler` uses: global user and per p
 
     To set up `wrangler` to work with your Cloudflare user, use the following commands:
 
-    - 🔧 `config`: an interactive command that asks you to pass your `email` and `api` key. Alternatively, 
-       you can use the flags `--email` and `--api-key` to the command to skip the interactive part.
+    - 🔧 `config`: an interactive command that asks you to pass your `
+    ` and `api` key.
```
Comment (xtuc) [inline README.md]: > asks you to pass your ``

I think it's missing something here?

File: README.md [hunk 2]
```diff
@@ -103,6 +103,12 @@ There are two types of configuration that `wrangler` uses: global user and per p
     $ brew install openssl@1.1
     ```
 
+1. Update
+    To get the latest version of Wrangler run:
+  
+    ```bash
+    cargo install --git https://github.com/cloudflare/wrangler --force
+    ```
```
Comment (xtuc) [inline README.md]: I'm not sure but `npm install @cloudflare/wrangler` should do the trick too because it's taking the latest release on GitHub

Comment (victoriabernard92) [reply README.md]: Are there different workflows for someone using cargo ? Or is everyone supposed to use npm now?

Comment (xtuc) [reply README.md]: We should keep both workflows (cargo or npm).

**Comment by signalnerve** [comment]
Closed in favor of #287

**Pull request closed without merging**